doc/getting_started/setup_verilator.md

```diff
@@ -151,15 +151,15 @@ The simulation includes a DPI module to map general-purpose I/O (GPIO) pins to t
 Observe the `gpio0-read` file for outputs (in the same directory as the trace):
 
 ```console
-$ cat gpio0-read
+cat gpio0-read
```

Comment:
There's another to remove on line 143 (Github won't let you comment on lines that are too far away from the actual changes)

Reply:
Good find! I've pushed a commit to fix that as well.